airgun/views/rhai.py

```diff
@@ -0,0 +1,138 @@
+from widgetastic.widget import (
+    Checkbox,
+    GenericLocatorWidget,
+    ParametrizedView,
+    Table,
+    Text,
+    TextInput,
+    View
+)
+
+from airgun.views.common import BaseLoggedInView
+
+
+class Overview(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+
+    @View.nested
+    class action_summary(View):
+        title = Text(".//h2[@class='title']")
+
+    @View.nested
+    class newest_systems(View):
+        title = Text(".//h2[@class='title']")
+        table = Table(".//section[contains(normalize-space(.),"
+                      "'Newest Systems')]//table")
+
+    @View.nested
+    class optimize_your_expeirience(View):
+        title = Text(".//h2[@class='title']")
+        registered_systems = Text(".//div[@class='chart-metrics']/span")
+        stability_issues = Text(
+            ".//span[normalize-space(.)='View stability issues']"
+            "/../../div[contains(@class, 'count')]")
+        security_issues = Text(
+            ".//span[normalize-space(.)='View security issues']"
+            "/../../div[contains(@class, 'count')]")
+
+
+class Actions(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+    issues = Text(".//div[@class='num']")
+    export_csv = Button("Export CSV")
+
+
+class InventoryAllHosts(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+    search = TextInput(locator=".//input[@placeholder='Find a system']")
+    actions = Dropdown("Actions")
+    systems_count = Text(".//h3[@class='system-count']")
+    table = Table(".//table",
+                  column_widgets={0: Checkbox(locator=".//input")})
+
+    @property
+    def is_displayed(self):
+        return (
+            self.title.text == "Inventory" and
+            self.menu.currently_selected == ["Insights", "Inventory"]
+        )
+
+
+class InventoryHostDetails(View):
+    hostname = Text(".//div[@class='modal-title']/h2//span")
+    close = GenericLocatorWidget(".//div[contains(@class, 'fa-close'])")
+
+
+class Rules(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+
+
+class Planner(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+    create_plan = GenericLocatorWidget(".//a[@class='create-plan']")
+
+    @ParametrizedView.nested
+    class plan(ParametrizedView):
+        PARAMETERS = ("plan_name", )
+        ROOT = ParametrizedLocator(
+            ".//h2[contains(normalize-space(.), {plan_name|quote})]/"
+            "ancestor::div[contains(@id, 'maintenance-plan')]")
+
+        title = Text(".//h2[@class='title']")
+        delete = GenericLocatorWidget(".//i[@tooltip='Delete this plan']")
+        edit = GenericLocatorWidget(".//i[@tooltip='Click to edit this plan']")
+        run_playbook = Button("Run Playbook")
+        export_csv = Button("Export CSV")
+        add_actions = Button("Add actions")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.title.text == "Planner" and
```

Comment:
same here, `title.is_displayed` should be enough

Reply:
Nope